lstchain/pointing/pointings.py

```diff
@@ -65,10 +65,9 @@ def cal_pointingposition(self, ev_time):
             a container filled with drive information
         """
         drive_container = LSTDriveContainer()
-        data = self._read_drive_report()
-        drive_container.time = data['time'].data
-        drive_container.azimuth_avg = data['azimuth_avg'].data
-        drive_container.altitude_avg = data['altitude_avg'].data
+        drive_container.time = drive_data['time'].data
+        drive_container.azimuth_avg = drive_data['azimuth_avg'].data
+        drive_container.altitude_avg = drive_data['altitude_avg'].data
```

Comment:
do you think it would be worth reading the other data as well?
It could be used later to check that the pointing was not crazy... What do you think?

Reply:
> do you think it would be worth reading the other data as well?
It could be used later to check that the pointing was not crazy... What do you think?

Yes, off course that would be good. In fact, the `drive_data` stores all information from the drive report. Only some of them are tagged as e,g. `time`, `azimuth_avg`, etc. Today we saw that the RA and DEC are also present in the Drive report. At present, there are no RA and DEC information in the DL1 files. They are indeed required for  finer analysis and can be added as metadata in the DL1 file.